src/js/grapl-cdk/bin/grapl-cdk.ts

```diff
@@ -5,22 +5,39 @@ import * as cdk from '@aws-cdk/core';
 import { GraplCdkStack } from '../lib/grapl-cdk-stack';
 import { EngagementUx } from '../lib/engagement';
 
-// Deployment parameters
-const deployName = undefined; // ex: 'Grapl-my-deployment' - GRAPL_CDK_DEPLOYMENT_NAME
-const graplVersion = undefined; // defaults to 'latest' - GRAPL_VERSION
-const watchfulEmail = undefined; // (optional) ex: ops@example.com - GRAPL_CDK_WATCHFUL_EMAIL
+// Either replace the `undefined`s or specify the environment variables.
+class DeploymentParameters {
+    readonly deployName = undefined  // ex: 'Grapl-my-deployment'
+        || process.env.GRAPL_CDK_DEPLOYMENT_NAME; 
 
-const stackName = process.env.GRAPL_CDK_DEPLOYMENT_NAME || deployName;
+    readonly graplVersion = undefined 
+        || process.env.GRAPL_VERSION 
+        || "latest";
+
+    readonly watchfulEmail = undefined  // (optional) ex: ops@example.com
+        || process.env.GRAPL_CDK_WATCHFUL_EMAIL; 
+
+    readonly operationalAlarmsEmail = undefined  // (optional) ex: ops-alarms@example.com
+        || process.env.GRAPL_CDK_OPERATIONAL_ALARMS_EMAIL; 
+
+    readonly securityAlarmsEmail = undefined  // (optional) ex: security-alarms@example.com
+        || process.env.GRAPL_CDK_SECURITY_ALARMS_EMAIL; 
```

Comment:
This all looks good, though if the reorder is deliberate we'll need to remove the line in the README that says "The environment variables take precedence over the values in bin/grapl-cdk.ts."

Reply:
I ended up separating out "here's where you hardcode stuff" and "here's where you get a concrete instance"